Updated for opt-out (#2749)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/fledge/index.md

removed when users clear their site data.

## How can I opt out of FLEDGE? {: #opt-out}

You can block access to the FLEDGE API either as a site owner, or as an individual user.

### How can sites control access? {: #opt-out-site}

> FLEDGE will eventually require sites to set permissions policy to allow FLEDGE functionality to be
> available. This will help ensure that arbitrary third parties can't use the API without a site's

Reviewer: ```suggestion
FLEDGE will eventually require sites to set a [Permissions Policy](/docs/privacy-sandbox/permissions-policy/)
to allow FLEDGE functionality to be available. This will help ensure that arbitrary third parties can't use the API without a site's
```

knowledge. However, to facilitate testing during [the first origin trial](/blog/privacy-sandbox-unified-origin-trial),

> this requirement is being [waived by default](https://github.com/WICG/turtledove/blob/main/Proposed_First_FLEDGE_OT_Details.md#permissions-policy).

Reviewer: ```suggestion
this requirement is [waived by default](https://github.com/WICG/turtledove/blob/main/Proposed_First_FLEDGE_OT_Details.md#permissions-policy).
```

Sites that would like to explicitly disable FLEDGE functionality during the testing period can use

> the relevant permissions policy to block access.

Reviewer: Capitalized as it's referencing the specific code item to set, versus a more general idea of permissions policies.

There are two FLEDGE permissions policies that can be set independently:
* `join-ad-interest-group` enables/disables functionality to add a browser to interest groups
* `run-ad-auction` enables/disables functionality to run an on-device auction

Access to FLEDGE APIs can be disabled completely in first-party contexts by specifying the following

> permissions policy in an HTTP response header:

Reviewer: ```suggestion
`Permissions-Policy` in an HTTP response header:
```

Reviewer: I could be convinced styling this copy as code is unnecessary. The logic was that this reference is about the HTTP header specifically.

Author: Good catch!

``` text
Permissions-Policy: join-ad-interest-group=(), run-ad-auction=()
```

> Usage of the APIs in an  iframe can be disabled by adding the following `allow` attribute to an

Reviewer: ```suggestion
You can disable usage of the APIs in an iframe by adding the following `allow` attribute to an
```

iframe element:

``` html
<iframe src="https://example.com" allow="join-ad-interest-group 'none'; run-ad-auction 'none'"></iframe>
```

> The Proposed First FLEDGE Origin Trial [Permissions-Policy](https://github.com/WICG/turtledove/blob/main/Proposed_First_FLEDGE_OT_Details.md#permissions-policy) section provides more detail.one

Reviewer: ```suggestion
The [Proposed First FLEDGE Origin Trial Permissions-Policy](https://github.com/WICG/turtledove/blob/main/Proposed_First_FLEDGE_OT_Details.md#permissions-policy) section provides more detail.
```
